src/js/angular/clustermanagement/templates/cluster-list.html

```diff
@@ -125,9 +126,14 @@
                     </div>
                 </td>
                 <td class="info-cell data">
-                    <div ng-if="node.item.isLocal">
-                        {{'cluster_management.update_cluster_group_dialog.current_node' | translate}}
-                    </div>
+                    <span ng-if="node.isLocal">
+                        {{'cluster_management.update_cluster_group_dialog.current_node' | translate}}<span ng-if="node.item.nodeState">, </span>
```

Comment:
umm, isn't it possible the comma to be added in the span below without ng-if?

Reply:
No, because the span below can exist without this one